### DISCUSSION TITLE
Record stream?

### DISCUSSION THREAD

**Discussion by f520k510** [GerryDazoo/Slinger#36] [Ideas]
I wonder if we can record stream with slinger?
Or will the old kmttg continue to work beyond that point?

thanks.

**Comment by GerryDazoo** [comment]
I thought I had replied via email. any ho, yes, the old kmttg should still work.

**Reply by GerryDazoo** [reply]
this also works... use curl to pipe the stteam to a file and then you can play it back with VLC etc.
>curl yourip:yourport/slingbox > c:\temp\slingbox.vid

**Reply by f520k510** [reply]
Thanks.

**Comment by sp222222** [comment]
you can hit the record button on VLC itself....that simple :) of course not sure about scheduling but i'm sure there's a vlc cmd line that can do that.

**Comment by TechTechAlways** [comment]
The best way to record is add sling server's address in the video source in OBS and record from there and there are tons of ways to customize OBS. OBS can be installed on the same machine where sling server runs.

**Comment by GerryDazoo** [comment]
The old kmttg should still work.

 

From: f520k510 ***@***.***> 
Sent: Monday, September 05, 2022 1:15 PM
To: GerryDazoo/Slinger ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GerryDazoo/Slinger] Record stream? (Discussion #36)

 

I wonder if we can record stream with slinger?
Or will the old kmttg continue to work beyond that point?

thaks.
